CMSgov/mint-app
# [EASI-2586] Add 'ARCHITECT' to TeamRoles for Collaborators

# EASI-2586

## Changes and Description

- 'ARCHITECT' added to GQL
- 'ARCHITECT' added to SQL
- 'Architect' added to front-end



## How to test this change

- Create a new model plan
- Add a collaborator
  - Confirm the 'Architect' selection is available in team roles
  - Select the 'Architect' role
- Confirm the collaborator was successfully added
- Confirm that the new entry in the 'plan_collaborators' table has the role 'ARCHITECT'

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
